README.md

```diff
@@ -13,6 +13,14 @@ This repository hosts the code for [IntelliJ IDEA](https://www.jetbrains.com/ide
 
 The project is developed under [Aalto Le-Tech research group](https://research.cs.aalto.fi/LeTech/) and is steered by **[@jsorva](https://github.com/jsorva)** and **[@oseppala](https://github.com/oseppala)**.
 
+## Code of Conduct
+
+The team follows Aalto University general [Code of Conduct](https://www.aalto.fi/en/aalto-university/code-of-conduct) principles.
```

Comment:
Nit: Is there link to this that is available for non-Aalto users?

Reply:
What do you mean? I think that link works for everyone

Reply:
Oh sorry, now I see it does. I only assumed it was restricted as it took me to login page but I guess that was because I was previously logged in but the session had timed out. For non-logged-in users, it apparently goes right to the page.